test/test.c

```diff
@@ -73,7 +75,7 @@ int test_files_match(char *example_name, char *expected_file_path,
 }
 
 int test_render(char *example) {
-  char *example_dir = "test/example/";
+  char *example_dir = TEST_EXAMPLE_DIR;
```

Comment:
This makes tests no longer crash if they are not started from the test directory.